src/docs/asciidoc/installing_upgrading.adoc

```diff
@@ -44,7 +44,7 @@ of the link:https://github.com/hazelcast/hazelcast-command-line[Command Line Int
 === Maven
 
 NOTE: As a prerequisite, make sure you have Java installed on your system.
-If you're using JDK 9 and newer, see <<running-in-modular-java>>.
+If you're using JDK 11 or later, see <<running-in-modular-java>>.
```

Comment:
Why did you change the JDK version to 11? It's actually true of the 9.

Reply:
Ah I see. We now support only the LTS releases. Therefore, the Java 9 version in the "Running in Modular Java" should be changed as well.